project/ScalazBuild.scala

```diff
@@ -4,13 +4,12 @@ import Keys._
 object Scalaz extends Build {
   val testDeps = Seq("org.scalacheck" %% "scalacheck" % "1.13.4" % "test")
 
-  def module(prjName: String) = Project(
-    id = prjName,
-    base = file(prjName)).settings(
+  def stdSettings(prjName: String) = Seq(
     name := s"scalaz-$prjName",
-    scalaVersion := "2.11.11",
+    scalaVersion := "2.12.2",
     scalacOptions ++= Seq("-feature","-deprecation", "-Xlint", "-language:higherKinds",
-                          "-Ydelambdafy:method", "-Ypartial-unification", "-target:jvm-1.8"),
+                          "-Ydelambdafy:method", "-Ypartial-unification", "-target:jvm-1.8", "-opt:l:project",
```

Comment:
it looks like `opt:l:project` is now deprecated: https://github.com/scala/scala/pull/5964